src/main/java/seedu/duke/util/StringConstants.java

```diff
@@ -237,7 +237,7 @@
             + "Aborting load...";
     public static final String ERROR_INVALID_CONFIG_VALUE = "Config \"%s\" has illegal value \"%s\". Aborting load...";
     public static final String ERROR_INVALID_TASK_DATA = "Invalid task data found in loaded data. Aborting load...";
-
+    public static final String ERROR_GRADE_REMOVAL = "Your grade has not been entered yet for this module...";
```

Comment:
I suggest putting as something like`ERROR_GRADE_REMOVAL_FAILED =  "Sorry, your grade has not been entered yet for this module. \nPlease first add some grades!";` to make the error msg more consistent

Reply:
I dont think it sounds right to say `Please first add some grades!`? because the user wants to remove a grade, so suggesting the user to add a grade just to remove it sounds redundant